File: 1291-sequential-digits/1291-sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int> v;
        for(int i=1;i<=9;i++){
            int num=i;
            int nxtdigit=i+1;
            while(num<=high && nxtdigit<=9){
                num=num*10+nxtdigit;
                if(num>=low && num<=high) v.push_back(num);
                ++nxtdigit;
            }
        }
        sort(v.begin(),v.end());
        return v;
    }
};

File: 1291-sequential-digits/1291-sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        v=[]
        c="123456789"
        for i in range(len(c)):
            for j in range(i+1,len(c)+1):
                curr=int(c[i:j])
                if low<=curr<=high:
                    v.append(curr)
        v.sort()
        return v
           